metadata-ingestion/tests/unit/test_transform_dataset.py

```diff
@@ -225,20 +225,7 @@ def test_simple_dataset_ownership_transformation(mock_time):
 
```

Comment:
Shouldn't we add similar test but for `OwnershipTransformer` so that we know the `not_a_dataset` is actually transformed in such case?

Reply:
done